app/src/layouts/tabular/index.ts

```diff
@@ -234,7 +236,7 @@ export default defineLayout<LayoutOptions, LayoutQuery>({
 							const arrayField = fieldsStore.getField(collection.value!, fieldParts.slice(0, -1).join('.'));
 
 							// Special case for translations to render the nested data in the translations display instead of the default display
-							if (types.at(-1) === 'o2m' && arrayField?.meta?.special?.includes('translations')) {
+							if (types.at(-1) === 'o2m' && arrayField?.meta?.display === 'translations') {
```

Comment:
We can't rely on the name of an extension for logic. Displays are extension types so the name could be anything